apps/src/weblab/WebLab.js

```diff
@@ -161,7 +160,10 @@ WebLab.prototype.init = function (config) {
   }
 
   function onAddFileImage() {
-    dashboard.assets.showAssetManager(null, 'image', this.loadCurrentAssets.bind(this), !this.studioApp_.reduxStore.getState().pageConstants.is13Plus);
+    dashboard.assets.showAssetManager(null, 'image', this.loadFileEntries.bind(this), {
+      showUnderageWarning: !this.studioApp_.reduxStore.getState().pageConstants.is13Plus,
+      useFilesApi: true
```

Comment:
It looks like we're setting `useFilesApi` true in two separate places (here and [line 124](https://github.com/code-dot-org/code-dot-org/pull/11041/files#diff-fc20c8d4ce1f48b628e41e82d1e6f117R124) just above).  Can we reduce this to a single source of truth?

More generally, is adding this flag to the `config` object the best way to configure WebLab to have this different behavior?  It might be, I'm just making sure we think about it.  Other options:

* We use this in a lot of React components.  Should this flag be in the `pageConstants` section of the Redux store?
* Instead of a flag, could we construct something with a common interface to put on the `config` object, so that fewer calling sites care whether we're using the Files API or not?
* Similarly, maybe for now we configure the clientApi.js module to export said interface depending on the app (for now?)

Just brainstorming.  Let me know what you think.